portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -53,8 +66,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String text = (String) entity.getProperty("comment");
       String username = (String) entity.getProperty("username");
       String email = (String) entity.getProperty("email");
+      String image = (String) entity.getProperty("image");
       String comment = username + " (" + email + ") said:" 
-        + "\n" + "\"" + text + "\"\n";
+        + "\n" + "\"" + text + "\"\n" + "<br/> <img src=\"" + image + "\" /> <br/>";
```

Comment:
Usually it's not recommended to append many strings. It is not much readable.

Consider using String.format("%s (%s) said: \n\"%s\"<br/><img src=\"%s\"/> <br/>", username, email, text, image);

(It's not quite clear either, but should be slightly better).